src/configurations/rebuttal.js

```diff
@@ -0,0 +1,12 @@
+let rebuttalConfig;
+
+
+rebuttalConfig = {
```

Comment:
Please add this inside the `debateConfig`.

Reply:
done